fsspec/implementations/sftp.py

```diff
@@ -77,18 +77,18 @@ def mkdir(self, path, create_parents=False, mode=511):
 
     def makedirs(self, path, exist_ok=False, mode=511):
         if self.exists(path) and not exist_ok:
-            raise FileExistsError("File exists: {}".format(path))
+            raise FileExistsError(f"File exists: {path}")
 
         parts = path.split("/")
         path = ""
 
         for part in parts:
-            path += "/" + part
+            path += f"/{part}"
             if not self.exists(path):
                 self.ftp.mkdir(path, mode)
 
     def rmdir(self, path):
-        logger.debug("Removing folder %s" % path)
+        logger.debug(f"Removing folder {path}")
```

Comment:
In fact, for logging it it recommended to use % notation and a list of string arguments (NOT % format) so that the string doesn't need to be evaluated at all if it's not to be rendered.
```suggestion
        logger.debug(f"Removing folder %s", path)
```

Where the argument is a string as opposed to something harder to evaluate, it makes negligible difference...

Reply:
There are a few other cases of this.